tdrs-backend/tdpservice/users/api/login.py

```diff
@@ -104,6 +107,10 @@ def decode_jwt(payload, issuer, audience, cert_sr, options=None):
     def get_auth_options(self, access_token: Optional[str], sub: Optional[str]) -> Dict[str, str]:
         """Set auth options to handle payloads appropriately."""
 
+    def handle_email(self, email):
```

Comment:
I want to change the function name but am open to ideas as I don't feel this describes what we are accomplishing in a helpful way.